plugin_intellij/src/main/java/org/xtclang/sdk/language/EcstasyFileType.java

```diff
@@ -0,0 +1,45 @@
+package org.xtclang.sdk.language;
+
+import com.intellij.openapi.fileTypes.LanguageFileType;
```

Comment:
Please take at any .java in javatools project to seeur coding conventions (e.g. org.xvm.compiler.Compiler.java). We are usually separating the import packages with a new line, trying not to use wildcard imports unless it's more than 20 imports from the same package and comment anything that is not absolutely obvious.